src/main/scala/is/hail/annotations/StagedRegionValueBuilder.scala

```diff
@@ -108,6 +108,19 @@ class StagedRegionValueBuilder private(val mb: MethodBuilder, val typ: Type, var
 
   def addDouble(v: Code[Double]): Code[Unit] = region.storeDouble(currentOffset, v)
 
+  def allocateBinary(n: Code[Int]): Code[Long] = {
```

Comment:
I'm pretty sure that this (and the `emitBinary` case down below) is already handled by `addBinary`, no?

Reply:
I don't have the bytes yet so I can't call addBinary, I'm going to get them from the InputStream via readBytes.  readBytes reads directly into the region.

Reply:
womp, right.